packages/number-input/src/Arrows/Arrow.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import Icon from '@leafygreen-ui/icon';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { Direction } from '../NumberInput/NumberInput.types';
+
+import {
+  arrowBaseStyles,
+  arrowThemeStyles,
+  downArrowRotateStyles,
+} from './Arrows.styles';
+import { ArrowProps } from './Arrows.types';
+
+/**
+ * @internal
+ */
+
+export const Arrow = ({
+  disabled,
+  handleClick,
+  handleArrowKeyDown,
+  direction,
+}: ArrowProps) => {
+  const { theme } = useDarkMode();
+
+  return (
+    <button
+      aria-label={`${direction} number`}
+      onClick={() => handleClick(direction)}
```

Comment:
I think it would be more appropriate to call the prop `onClick` and to have a `const handleClick = () => {...}` defined outside of the render function